app/services/unsplash_service.rb

```diff
@@ -0,0 +1,18 @@
+class UnsplashService
+  def get_image(location)
+    response = conn.get("/search/photos") do |req|
+      req.params["query"] = location
+    end
+    json = JSON.parse(response.body, symbolize_names: true)
+    json[:results][0][:urls][:full]
```

Comment:
Customized response to receive "Full" image. Other choices were small, thumbnail, regular, and other.